packages/eos-components/src/components/Header.vue

```diff
@@ -1,7 +1,7 @@
 <template>
   <b-navbar class="header" :class="{ shadow: hasScrolled }" fixed="top">
     <b-container class="px-3">
-      <img class="logo" :src="logo" @click="$emit('click-logo')">
+      <img v-if="showLogo" class="logo" :src="logo" @click="$emit('click-logo')">
```

Comment:
How could users navigate back to the landing page if they can't click on the logo? It's the only way to go back.